server/server/src/account_service.rs

```diff
@@ -279,8 +280,37 @@ pub async fn admin_get_account_info(
             }
         });
 
+    let usage: u64 = db
+        .transaction(|tx| get_usage_helper(tx, &owner.0))?
+        .map_err(|err| internal!("Cannot find user's usage, owner: {:?}, err: {:?}", owner, err))?
+        .iter()
+        .map(|a| a.size_bytes)
+        .sum();
+
+    let usage_str = bytes_to_human(usage);
+
+    let mut last_active = 0;
+
+    if let Some(ids) = db.owned_files.get(&owner)? {
+        for id in ids {
+            let metadata = db.metas.get(&id)?.ok_or_else(|| {
+                internal!("Metadata for id should exist, id: {:?}, owner: {:?}", id, owner)
+            })?;
+
+            if metadata.version > last_active {
+                last_active = metadata.version
```

Comment:
This won't capture modifications to shared files, I think we actually can't capture this field without storing data on the server, so I'm down to remove `last_active` from this entirely for now.